Packs/Sixgill-Darkfeed/Integrations/Sixgill_DVE_Feed/Sixgill_DVE_Feed.yml

```
@@ -0,0 +1,120 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Sixgill_DVE_Feed
+  version: -1
+configuration:
+- display: Client Id
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret
+  hidden: false
+  name: client_secret
+  required: true
+  type: 4
+- defaultvalue: '1000'
+  hidden: false
+  name: maxIndicators
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: '2'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- display: Bypass exclusion list
```

```suggestion
- additionalinfo: When selected, the exclusion list is ignored for indicators from this
    feed. This means that if an indicator from this feed is on the exclusion list, the
    indicator might still be added to the system. 
display: Bypass exclusion list
```

Please add this additional info key in order to fix the validation error.
This parameter should look like this: 
```
- additionalinfo: When selected, the exclusion list is ignored for indicators from this
    feed. This means that if an indicator from this feed is on the exclusion list, the
    indicator might still be added to the system.
  display: Bypass exclusion list
  name: feedBypassExclusionList
  required: false
  type: 8```